### PR TITLE
Ana/add message for polkadot treasuries

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Adds this message to Polkado Treasury proposals:

image

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f68ba726a02431af7b7250f--lunieio.netlify.app

**Comment by jbibla** [comment]
this is not the right way to do this - if we want to include the message should alter the summary. the problem is that the summary is being displayed as a param change proposal - not a `type: treasury`

**Review by jbibla — CHANGES_REQUESTED** [review]
not important right now and not the right implementation

**Comment by Bitcoinera** [comment]
> this is not the right way to do this - if we want to include the message should alter the summary. the problem is that the summary is being displayed as a param change proposal - not a `type: treasury`

No, the problem is that we would need a network specific message for Polkadot networks, since we don't know how other Treasury proposals in other networks will look like. Council only exists in Polkadot networks

**Comment by jbibla** [comment]
true. so in the API we can change the summary message based on whether or not it's a polkadot network.

**Comment by Bitcoinera** [comment]
> true. so in the API we can change the summary message based on whether or not it's a polkadot network. but we shouldn't use a bar for this.

Done!

**Comment by Bitcoinera** [comment]
Debugging this:

image

File: api/lib/reducers/polkadotV0-reducers.js [hunk 1]
```diff
@@ -607,7 +607,9 @@ function treasuryProposalReducer(
     deposit: toViewDenom(network, Number(proposal.deposit)),
     proposer: proposal.proposer,
     beneficiary: proposal.beneficiary, // the account getting the tip
-    summary: getProposalSummary(proposalTypeEnum.TREASURY),
+    summary:
+      getProposalSummary(proposalTypeEnum.TREASURY) +
+      `\nTreasury proposals in ${network.title} can only be voted by the council members.`,
```
Comment (jbibla) [inline api/lib/reducers/polkadotV0-reducers.js:612]:
```suggestion
      `\nTreasury proposals on ${network.title} can only be voted on by council members.`,
```

**Review by jbibla — APPROVED** [review]

File: api/lib/reducers/polkadotV0-reducers.js (hunk 1 above)
Comment (Bitcoinera) [reply api/lib/reducers/polkadotV0-reducers.js]: Native speaker powah!

**Pull request merged**